test/unit.js

```diff
@@ -82,69 +82,40 @@ metatests.test('Correct cell values', async (test) => {
 metatests.test('Prevent arbitrary js code execution', async (test) => {
   const sheet = new Sheet();
 
+  sheet.on('error', (error) => {
+    test.strictSame(error.constructor.name === 'TypeError', true);
+  });
+
```

Comment:
You can't change try/catch to on('error') in this case, because all cases when metacals will not generate errors but we expect error will not be found (will be just silently skipped). Solution: we need to plan exact number of expected errors and should implement counter or use test runner API to count errors.